lib/al/Library/Matrix/MatrixUtil.h

```diff
@@ -1,21 +1,30 @@
 #pragma once
 
 #include <math/seadMatrix.h>
-#include <math/seadQuat.h>
 #include <math/seadVector.h>
 
 namespace nn::util::neon {
-struct MatrixColumnMajor4x3fType;
-struct MatrixColumnMajor4x4fType;
+class MatrixColumnMajor4x3fType;
+class MatrixColumnMajor4x4fType;
```

Comment:
Any reason why these should be classes instead of structs?

Reply:
I wouldn't know. Changing them to struct